lib/fastlane/plugin/emerge/helper/github.rb

```diff
@@ -43,10 +43,7 @@ def self.previous_sha
         if is_push?
           github_event_data.dig(:before)
         else
-          shell_command = "git rev-parse HEAD^"
-          UI.command(shell_command)
-          stdout, _, status = Open3.capture3(shell_command)
-          stdout.strip if status.success?
+          github_event_data.dig(:before) || self.base_sha
```

Comment:
Why the `|| self.base_sha`? When would using base_sha be correct?

Reply:
`before` is empty for the first commit of a PR. In that base, base_sha is correct. @rbro112 had to do the same thing for our gradle plugin